modules/url-events.js

```diff
@@ -0,0 +1,107 @@
+const Bitly = require('bitly')
+const cheerio = require('cheerio')
+const fs = require('fs')
+const request = require('request')
+
+var LOG = {}
+
+var LOGFILE = __rootdir + '/data/ofn.json'
+try {
+  LOG = require(LOGFILE)
+} catch (e) {
+  //
+}
+
+var getTitle = function (url) {
+  return new Promise(
+    (resolve, reject) => {
+      const maxSize = 1024
+      var size = 0
+      var response = ''
+      var req = request(url)
```

Comment:
Can this use `request-promise`?

Reply:
It quite possibly can, but I tried and never found a way to monitor the download size.